src/common/actions/repositories.js

```diff
@@ -0,0 +1,68 @@
+import 'isomorphic-fetch';
+
+import conf from '../helpers/config';
+
+const BASE_URL = conf.get('BASE_URL');
+
+export const FETCH_REPOSITORIES = 'FETCH_REPOSITORIES';
+export const FETCH_REPOSITORIES_ERROR = 'FETCH_REPOSITORIES_ERROR';
```

Comment:
My reading of FSA's is that you only need the  FETCH_REPOSITORIES action, FETCH_REPOSITORIES_ERROR is replaced by the optional error property of FETCH_REPOSITORIES